server/src/modules/mentorship-request/mentorship-request.model.ts

```diff
@@ -14,45 +12,33 @@ import { MentorshipResponse } from '../mentorship-response/mentorship-response.m
 
 @Table({
   tableName: 'mentorship_request',
+  timestamps: true,
```

Comment:
Nice, I like timestamps on tables in general, can make debugging or solving real-world issues much easier (or even cleaning up corrupted data, don't ask me how I know...)